functorch/experimental/_map.py

```diff
@@ -118,12 +230,12 @@ def map_functionalize(interpreter, f, xs, *args):
                 "torch.map is aliasing the input!"
             )
 
-        map_return = map(functional_map_fn, unwrapped_xs, *unwrapped_args)
+        map_return = map_impl(functional_map_fn, *inputs)
```

Comment:
This is gonna merge conflict with https://github.com/pytorch/pytorch/pull/99461

Reply:
Will do a rebase.